webapp/src/client/client.js

```diff
@@ -11,14 +11,13 @@ export default class Client {
         this.url = url + '/plugins/' + id;
     }
 
-    startMeeting = async (
-        channelId, personal = true, topic = '', meetingId = 0, force = false,
-    ) => {
+    startMeeting = async (channelId, rootId, personal = true, topic = '', meetingId = 0, force = false) => {
```

Comment:
Is `meetingId` ever defined when calling these client functions?

Reply:
@mickmister I don't think this will ever be defined.

Reply:
@raghavaggarwal2308 If it's never defined when this function is called anywhere in the plugin, then I think we should remove the function parameter in this PR. Do you think it will eventually be used?

Reply:
@mickmister I don't think it will be used. Removed it now.